src/components/confirmDelete/ConfirmDelete.jsx

```diff
@@ -1,41 +1,76 @@
-import './ConfirmDelete.css'
+import "./ConfirmDelete.css";
 import { useParams, Link, useNavigate } from "react-router-dom";
-import React,{useCallback} from 'react';
-import axios from 'axios';
-import { useDispatch, useSelector } from 'react-redux';
-import { deleteInvoice } from '../../invoiceSlice/InvoiceSlice';
+import React, { useCallback } from "react";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+import { deleteInvoice } from "../../invoiceSlice/InvoiceSlice";
 
+const ConfirmDelete = ({ goBack }) => {
+  const dispatch = useDispatch();
+  const darkMode = useSelector((state) => state.invoice.isDarkMode);
 
-const ConfirmDelete = ({goBack}) => {
-  const dispatch = useDispatch()
-  const darkMode = useSelector((state) => state.invoice.isDarkMode)
-
-  const navigate = useNavigate()
+  const navigate = useNavigate();
   const { id } = useParams();
 
-  
+  const deleteInvoice = useCallback(async () => {
+    await axios.delete(`https://invoice-api-9l7b.onrender.com/invoice/${id}`);
```

Comment:
avoid repeating endpoint urls in code..
whenthe base url chnages, you will suffer the pain of changing in all components..
we mostly extract the baseURL into cofig file or .env